_sass/custom/_breadcrumbs.scss

```diff
@@ -0,0 +1,51 @@
+.breadcrumb {
+    margin-bottom: 0;
+    border-radius: 0;
+    border-top: 1px solid $neutral-90;
+    border-bottom: 1px solid $neutral-90;
+    align-items: center;
+    min-height: 3.5rem;
+    background-color: #fff;
+    font-size: 0.9rem;
+    color: $neutral-30;
+    box-shadow: 0px 1px 2px 0px rgba(0,0,0,0.3);
+
+    ul, ol {
+        margin-bottom: 0;
+        padding-left: 0;
+        list-style-type: "";
+        display: flex;
+        flex-wrap: wrap;
+        font-size: inherit;
+
+        li {
+            &:before {
+                color: inherit;
+            }
+            
+        }
+    }
+
+    .container {
+        max-width: 1376px;
```

Comment:
I guess here we need to think about responsible pages, but we can take care of this kind of related change in another PR. What do you think @lilyndk

Reply:
@dricazenck Yes, we can create another task for mobile breadcrumbs.